### ISSUE TITLE
Humble Feature request: Texture to geometry

### ISSUE THREAD

**Issue by melvixiodf** [mantissa-/RandoMesh#4]
Hi Mantissa, Hope you are doing well.

This is a great addon.
Any chance this is possible in Blender?

https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2019/ENU/Maya-Modeling/files/GUID-79763ED5-E7BD-408B-937B-92743EDDF57A-htm.html

With an id map texture, you can create a topology with edge boundaries around the colors specified in the maps. If the texture is tiled, the topology is created on the geo as it was tiled with the topology pattern being the same for each location of color.

I love your work. I am not sure why Blender doesn't have it. I have looked at Mesh lab but I don't think it is possible. I have also looked at inkscape, importing svg in Blender but that might be problematic if the mesh isn't flat.

This makes the creation of patterns on complex meshes simpler and straight forward. I am still wondering if triplanar mapping the texture would work before converting the texture to geometry.

I consider it very useful and wanted to humbly ask if it could be added to RandoMesh or a new addon if you feel that would be more appropriate.

Also thank you so much for sharing your tools with us. 

Thanks

**Comment by mantissa-** [comment]
I think this a bit out of the scope of this project, and if I'm honest, my current abilities. I know TexTools (https://github.com/SavMartin/TexTools-Blender) can generate ID passes, maybe look in to that?

**Comment by melvixiodf** [comment]
Thanks for replying back. And I completely understand. I sincerely apologize if I bothered you.

Unfortunately, I don't think textools can generate geometry with edge loops around different colors in an image. The goal is to generate geometry not ID passes from the colors in the image like the link above.

Currently, just an image plane converted to geometry would be sufficient because I found out doing this for a tiled image on a mesh would be too complex to compute.

Hope you have a nice day

All the best.
